Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[workspace]
+members = ["db", "cli"]
+resolver = "2"
+
+[workspace.dependencies]
+color-eyre = "0.6.3"
+sqlx = { version = "0.7", default-features = false }
+tokio = { version = "1", features = ["full"] }
+tracing = "0.1.40"
+url = "2.5.0"
+uuid = { version = "1.8.0", features = ["v4"] }
```

Comment:
These are dependencies that can be used by any crate in the workspace. It helps make sure all the crates we write are using the same versions of dependencies!